spec/requests/api/vms_spec.rb

```diff
@@ -177,6 +177,19 @@ def update_raw_power_state(state, *vms)
       expect_single_action_result(:success => true, :message => "starting", :href => :vm_url, :task => true)
     end
 
+    it "starting a vm queues it properly" do
+      api_basic_authorize action_identifier(:vms, :start)
+      update_raw_power_state("poweredOff", vm)
+
+      run_post(vm_url, gen_request(:start))
+
+      expect_single_action_result(:success => true, :message => "starting", :href => :vm_url, :task => true)
+      expect(MiqQueue.where(:class_name  => vm.class.name,
+                            :instance_id => vm.id,
+                            :method_name => "start",
+                            :zone        => zone.name).count).to eq(1)
+    end
+
```

Comment:
@abellotti Do you think we need to test the case where the request comes in to an appliance in one zone but should be queued in a different zone? Not really sure how necessary that test is because in either case the message should be queued to the zone of the object and this test does that. So I'll leave it up to you.